### PR TITLE
Clear contents of `log_text` after successfull backup

### LINKED ISSUE #1389: Clear contents of `logText` after successfull backup.
Currently the label `logText` is not cleared after a borg command finished. When creating a backup the label will show the path of the last backuped file even after backup completion.

Instead the `progressText` and `logText` labels should be cleared if they show information only relevant during the command execution.

### PR DESCRIPTION
**Pull request by aadityasinha-dotcom**
fixes #1389

### CODE REVIEW

**Comment by diivi** [comment]
Just wanted to add here that personally, I'd like to see the backup finished message (as a success feedback) and then maybe click on a "Done" or "Clear" button to clear this text. This is also why I opened #1618

**Comment by DaffyTheDuck** [comment]
> Just wanted to add here that personally, I'd like to see the backup finished message (as a success feedback) and then maybe click on a "Done" or "Clear" button to clear this text. This is also why I opened #1618. With the current approach, the user can easily miss the "backup finished" text.

Or I guess the other approach could be like adding some time constraint, after that the message will automatically disappear if the backup is successful, if there's failure then we could add a clear button to clear the log as you mentioned in #1618 because having a time constraint there will be a problem

**Comment by aadityasinha-dotcom** [comment]
Yep we can add time constraint

**Comment by aadityasinha-dotcom** [comment]
For me clear button can be a little hassle, we can add the clear button when we get some failure other wise it will show successful message for few seconds 😊

**Comment by real-yfprojects** [comment]
> With the current approach, the user can easily miss the "backup finished" text.

Are you talking about this PR or the current master branch?
I am not in favour of the timed approach because I could easily miss a message when I am away from the computer. My intention of #1389 was only clearing the log for progress/status updates that aren't relevant after the borg job has finished. This would include showing the added files when creating a backup which looks like this:

```
/home/user/Documents/new.txt (A)
```

File: src/vorta/borg/create.py [hunk 1]
```diff
@@ -36,6 +36,8 @@ def process_result(self, result):
                 self.app.backup_progress_event.emit(self.tr('Backup finished with warnings. See logs for details.'))
             else:
                 self.app.backup_progress_event.emit(self.tr('Backup finished.'))
+                self.app.backup_progress_event.emit(self.tr(''))
+                self.app.backup_log_event.emit('', {})
```
Comment (real-yfprojects) [inline src/vorta/borg/create.py:40]:
```suggestion
                self.app.backup_log_event.emit('', {})
```

I imagine that to do the trick. Is there any log event emitted with vital information?

Comment (real-yfprojects) [reply src/vorta/borg/create.py]: > Is there any log event emitted with vital information?

?

Comment (aadityasinha-dotcom) [reply src/vorta/borg/create.py]: no

**Review by real-yfprojects — DISMISSED** [review]

File: src/vorta/borg/create.py [hunk 2]
```diff
@@ -42,8 +42,9 @@ def process_result(self, result):
                     ).format(LOG_DIR.as_uri())
                 )
             else:
+                self.app.backup_progress_event.emit(self.tr('Backup finished.'))
+                self.app.backup_log_event.emit('', {})
                 self.app.backup_progress_event.emit(f"[{self.params['profile_name']}] {self.tr('Backup finished.')}")
-
```
Comment (real-yfprojects) [inline src/vorta/borg/create.py:46]: There must be an empty line between two methods.

**Review by real-yfprojects — CHANGES_REQUESTED** [review]

**Review by m3nu — DISMISSED** [review]

**Review by real-yfprojects — APPROVED** [review]

**Pull request merged**